Sources/Controllers/Cloud/DeleteAccountViewController.swift

```diff
@@ -133,7 +134,12 @@ final class DeleteAccountViewController: OABaseButtonsViewController, OAOnDelete
         if status == .notStarted {
             let deleteConfirmRow = infoSection.createNewRow()
             deleteConfirmRow.cellType = OASimpleTableViewCell.reuseIdentifier
-            deleteConfirmRow.title = localizedString("osmand_cloud_delete_account_descr")
+            let email = OAAppSettings.sharedManager().backupUserEmail.get() ?? ""
+            let deleteConfirmTitle = NSMutableAttributedString(string: String(format: localizedString("osmand_cloud_delete_account_descr_with_email"), email),
+                                                                attributes: [.font: UIFont.preferredFont(forTextStyle: .body)])
+            deleteConfirmTitle.addAttribute(.font, value: UIFont.preferredFont(forTextStyle: .headline),
```

Comment:
What will happen when the email is empty? Maybe it's better to add a check for the range, like this:
```
let formattedString = String(format: localizedString("osmand_cloud_delete_account_descr_with_email"), email)
let deleteConfirmTitle = NSMutableAttributedString(string: formattedString,
                                                    attributes: [.font: UIFont.preferredFont(forTextStyle: .body)])
if let emailRange = formattedString.range(of: email) {
    let range = NSRange(emailRange, in: formattedString)
    deleteConfirmTitle.addAttribute(.font, value: UIFont.preferredFont(forTextStyle: .headline), range: range)
}
```